core/templates/pages/exploration-editor-page/services/editor-first-time-events.service.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('Editor First Time Events Service', () => {
     sas = TestBed.get(SiteAnalyticsService);
   });
 
-  it('should', () => {
+  it('should message', () => {
```

Comment:
Can you add a more descriptive message here?

Reply:
Done.